package.json

```diff
@@ -52,27 +52,42 @@
     "prettier": "repo-tools exec -- prettier --write src/*.js src/*/*.js samples/*.js samples/*/*.js test/*.js test/*/*.js system-test/*.js system-test/*/*.js",
     "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
     "system-test": "repo-tools test run --cmd mocha -- system-test/*.js --no-timeouts --bail",
-    "test": "repo-tools test run --cmd mocha -- test/*.js"
+    "test": "repo-tools test run --cmd mocha -- test/*.js",
+    "check": "gts check",
+    "clean": "gts clean",
+    "compile": "tsc -p .",
+    "fix": "gts fix",
+    "prepare": "npm run compile",
+    "pretest": "npm run compile",
+    "posttest": "npm run check"
   },
   "dependencies": {
     "@google-cloud/common": "^0.13.0",
-    "@google-cloud/logging": "^1.0.0",
+    "@google-cloud/logging": "^1.1.1",
     "extend": "^3.0.0",
     "is": "^3.2.0",
-    "lodash.mapvalues": "^4.6.0"
+    "lodash.mapvalues": "^4.6.0",
+    "@types/lodash.mapvalues": "^4.6.3"
   },
   "devDependencies": {
     "@google-cloud/nodejs-repo-tools": "^2.1.0",
+    "@types/lodash.mapvalues": "^4.6.3",
+    "@types/extend": "^3.0.0",
+    "@types/is": "0.0.17",
+    "@types/node": "^8.0.47",
+    "@types/winston": "^2.3.7",
     "codecov": "^3.0.0",
     "eslint": "^4.9.0",
     "eslint-config-prettier": "^2.6.0",
     "eslint-plugin-node": "^5.2.0",
     "eslint-plugin-prettier": "^2.3.1",
+    "gts": "latest",
     "ink-docstrap": "^1.3.0",
     "jsdoc": "^3.5.5",
     "mocha": "^3.2.0",
     "prettier": "^1.7.4",
     "proxyquire": "^1.7.10",
+    "typescript": "^2.4.1",
```

Comment:
They make semver breaking changes :(  Pin to `~2.6.1`

Reply:
Done.